azkaban-web-server/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -611,7 +614,19 @@ private void ajaxScheduleCronFlow(final HttpServletRequest req,
       return;
     }
 
-    final DateTimeZone timezone = DateTimeZone.getDefault();
+    DateTimeZone timezone = DateTimeZone.getDefault();
+    if (hasParam(req, "timezone")) {
+      String timezoneParam = getParam(req, "timezone");
+      try {
+        timezone = DateTimeZone.forID(timezoneParam);
+      } catch (final Exception e) {
```

Comment:
It would be nice if you could specify exactly what Exception it is. Applies to both the cases.